src/react-apps/applications/ux-editor/src/containers/TempContainer.tsx

```diff
@@ -0,0 +1,278 @@
+import * as React from 'react';
```

Comment:
Is TempContainer still used? Sounds like something temporary ;)

Reply:
TempContainer is used in "DesignView", so we should probably sit down and agree on a name for this. I'll scooch over to you and Nina and ask about it after fixing the other comments.